src/pages/index.js

```diff
@@ -310,7 +309,7 @@ profileAvatarEditButton.addEventListener("click", () => {
   avatarImageFormValidator.toggleButtonState();
```

Comment:
`      userProfileAvatar.src = response.avatar;
`

* Class `UserInfo` should set the entire user’s data (including the **avatar**)